csrankings-b.csv

```diff
@@ -29,6 +29,7 @@ Bai Wang 0001,BUPT,https://scs.bupt.edu.cn/info/1088/1313.htm,s5INMX8AAAAJ
 Baihua Li,Loughborough University,https://www.lboro.ac.uk/departments/compsci/staff/academic-teaching/baihua-li,sQT_ID0AAAAJ
 Baihua Zheng,Singapore Management University,https://sis.smu.edu.sg/faculty/profile/9643/ZHENG-Baihua,o5k9wxUAAAAJ
 Bailin Deng,Cardiff University,https://www.cardiff.ac.uk/people/view/598849-deng-bailin,zNXnH-UAAAAJ
+Baiquan Su,BUPT,https://teacher.bupt.edu.cn/subaiquan/zh_CN/index.htm,E9Q_OuMAAAAJ

```

Comment:
Mechanical Engineering / Biomedical Engineering, not CS.

Reply:
Baiquan Su can advise Phd students in Robotics area. He published several papers in ICRA & IROS.

Baiquan Su, Jie Tang, Hongen Liao: Automatic laser ablation control algorithm for an novel endoscopic laser ablation end effector for precision neurosurgery. IROS 2015